rpm/mapplauncherd-booster-firejail.spec

```diff
@@ -0,0 +1,43 @@
+Name:       mapplauncherd-booster-firejail
+Summary:    Firejail application launcher
+Version:    0.0.1
+Release:    1
+License:    LGPLv2.1
+URL:        https://github.org/jolla/mapplauncherd-booster-firejail
+Source0:    %{name}-%{version}.tar.bz2
+BuildRequires:  pkgconfig(Qt5Core)
+BuildRequires:  pkgconfig(Qt5Concurrent)
+BuildRequires:  mapplauncherd-devel >= 4.1.0
+BuildRequires:  pkgconfig(qdeclarative5-boostable)
+Requires:  mapplauncherd >= 4.1.0
+Requires:  systemd-user-session-targets
+Requires:  firejail
+Requires:  xdg-dbus-proxy
+
+%description
+Proof-of-concept booster implementation for launching apps in a Firejail
+sandbox with optional privileges.
+
+%prep
+%setup -q -n %{name}-%{version}
```

Comment:
autosetup -p1 <= even though there are no patches

Reply:
Yeah that's a nice default